milabench/validation/validation.py

```diff
@@ -0,0 +1,128 @@
+import json
+from contextlib import contextmanager
+from dataclasses import dataclass, field
+
+from ..structs import BenchLogEntry
+
+
+class ValidationLayer:
+    """Validation layer interface, captures events, makes report"""
+
+    _rc = 0
```

Comment:
What does this do?

Reply:
this is the return code of the validation layer, this is used to make milabench fail on critical errors..

Reply:
This is a class variable, though, but grep tells me it's only ever accessed through self.

Reply:
It was to make the value available through inheritance without having to call the super.__init__ to avoid changing too much code but I guess it is not needed right now.
I removed it